### ISSUE TITLE
Can't install extensions since v18.09.

### ISSUE THREAD

**Issue by StefanD986** [boot2docker/boot2docker#1360]
Since the v18.09 release of boot2docker stallation of extensions via tce-load is broken.

The installation **fails when trying to mount the loopback device**.

### Steps to reproduce

```
$ docker-machine create --driver virtualbox default && docker-machine ssh
```
then on the boot2docker VM:
```
docker@default:~$ tce-load -wi nano
nano.tcz.dep OK
Downloading: ncurses5.tcz
Connecting to repo.tinycorelinux.net (89.22.99.37:80)
ncurses5.tcz         100% |***********************************************************************************************************************************************************|   196k  0:00:00 ETA
ncurses5.tcz: OK
mount: mounting /dev/loop0 on /tmp/tcloop/ncurses5 failed: No such device
```
The last line above shows the error

### Expected behavior
Executing the same steps with boot2docker v18.06 shows the expected behavior:

```
$ docker-machine create --driver virtualbox --virtualbox-boot2docker-url https://github.com/boot2docker/boot2docker/releases/download/v18.06.1-ce/boot2docker.iso default
(...)
$ docker-machine ssh
```
then in boot2docker v18.06 VM:
```
docker@default:~$ tce-load -wi nano
nano.tcz.dep OK
Downloading: ncurses5.tcz
Connecting to repo.tinycorelinux.net (89.22.99.37:80)
ncurses5.tcz         100% |***********************************************************************************************************************************************************|   196k  0:00:00 ETA
ncurses5.tcz: OK
Downloading: file.tcz
Connecting to repo.tinycorelinux.net (89.22.99.37:80)
file.tcz             100% |***********************************************************************************************************************************************************|   384k  0:00:00 ETA
file.tcz: OK
Downloading: nano.tcz
Connecting to repo.tinycorelinux.net (89.22.99.37:80)
nano.tcz             100% |***********************************************************************************************************************************************************|   144k  0:00:00 ETA
nano.tcz: OK
docker@default:~$ 
```

### System info
Operating System: OSX Mojave 10.14.1

```
$ docker --version
Docker version 18.09.0, build 4d60db4

$ docker-machine --version
docker-machine version 0.16.0, build 702c267f
```

**Comment by tianon** [comment]
I think this is root-caused by the missing `CONFIG_SQUASHFS`, and as such will be fixed by #1359. :+1:

In the meantime, you should be able to use `tce-load -wicl ...` instead (and it'll use `unsquashfs` to extract instead of mounting; see https://github.com/boot2docker/boot2docker/blob/9ca8f5bce3377787cc20a815f70b497a195a5760/files/tce-load.patch).

**Comment by tianon** [comment]
See https://github.com/boot2docker/boot2docker/releases/tag/v18.09.1-beta1 and https://github.com/boot2docker/boot2docker/releases/tag/v18.09.1-beta2 (which include this fix). :+1:
